packages/babel-parser/src/parser/expression.js

```diff
@@ -1208,12 +1232,52 @@ export default class ExpressionParser extends LValParser {
           }
         }
       }
+
       // fall through
       default:
         throw this.unexpected();
     }
   }
 
+  // https://github.com/js-choi/proposal-hack-pipes
+  parseTopicReference() {
+    const pipeProposal = this.getPluginOption("pipelineOperator", "proposal");
+
+    if (pipeProposal) {
```

Comment:
When can this be falsy?

Reply:
When someone uses `#` (or `%` in the future) on its own, outside of a pipe expression, and without having turned on the `pipelineOperator` plugin.

I can change the method name to `maybeParseTopicReference` and add a comment to this line, to make that clearer.

Reply:
Oh a comment would be enough 👍

Reply:
Addressed in fc176687852f984aeddf8814bdf24a58bbce0b47.